### ISSUE TITLE
[enhancement] A standardised way of pointing out files

### ISSUE THREAD

**Issue by mFIND** [Soulghost/iblessing#4]
For easy of use it would be nice if each scanner/generator would accept the same file path in parameter '-f', searching for the appropriate subfile if necessary (e.g. binary file)

Imho this enhancement is a low priority :)

**Comment by Soulghost** [comment]
Good idea! Maybe we can specify a new option (such as --key) and ignore the -f (just like other tools to specify the input file path at end), then we can search by key to match binary files, report files, etc.

Thank you so much for your suggestion, I will add it to the to-do list.
